@shared/constants/stellar.ts

```diff
@@ -47,3 +47,7 @@ export const DEFAULT_NETWORKS: Array<NetworkDetails> = [
   MAINNET_NETWORK_DETAILS,
   TESTNET_NETWORK_DETAILS,
 ];
+
+export const SOROBAN_RPC_URLS = {
```

Comment:
There isn't a public Soroban RPC at the moment so not sure what we can set for this in the time being, I use a local quickstart node during development,